sql/telemetry/retention_1_week/view.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE VIEW
+  `retention`
```

Comment:
This name should match the filename. And perhaps we should make this name desktop-specific? Perhaps `desktop_retention_1_week`?

Or, would we potentially be better off with `clients_last_seen_1_week_retention` or `clients_last_seen_day_13` or `clients_last_seen_active_last_week` where we do `SELECT * REPLACE(DATE_SUB(submission_date, INTERVAL 13 DAY) AS submission_date)` and add the `retained` column.

Reply:
`desktop_retention_1_week` sounds best of those options to me.